android/src/main/java/nl/fhict/happynews/android/persistence/ReadingHistoryContract.java

```diff
@@ -0,0 +1,33 @@
+package nl.fhict.happynews.android.persistence;
+
+import android.provider.BaseColumns;
+
+/**
+ * Created by Tobi on 08-May-17.
+ */
+public final class ReadingHistoryContract {
+    public static final  int    DATABASE_VERSION   = 1;
+    public static final  String DATABASE_NAME      = "database.db";
+    private static final String TEXT_TYPE          = " TEXT";
+    private static final String COMMA_SEP          = ",";
+
+    // To prevent someone from accidentally instantiating the contract class,
+    // make the constructor private.
+    private ReadingHistoryContract() {}
+
+    /* Inner class that defines the table contents */
+    public static class HistoryEntry implements BaseColumns {
+        public static final String TABLE_NAME       = "History";
+        public static final String COLUMN_NAME_COL1 = "Post_UUID";
```

Comment:
I would suggest renaming this to `COLUMN_POST_UUID` to better reflect the content.